src/uint.rs

```diff
@@ -184,6 +184,20 @@ impl<const LIMBS: usize> Uint<LIMBS> {
     pub const fn to_odd(self) -> ConstCtOption<Odd<Self>> {
         ConstCtOption::new(Odd(self), self.is_odd())
     }
+
+    /// Convert a single-limb `Uint` to a `u64` (const-friendly)
+    #[cfg(target_pointer_width = "32")]
+    pub const fn to_u64(self) -> u64 {
+        assert!(LIMBS == 1, "number of limbs must be 1");
```

Comment:
I did consider a `Uint<1>` implementation directly, but wasn't sure if that was preferable.

Anyway, agreed that it would be both safer and more idiomatic to do both of these.

Any preference as to where to place such a `From` implementation within the existing file structure, to keep things tidy?

Reply:
There's a `src/uint/from.rs` module which contains several conversions, although I suppose this is more of a `to`